osfoffline/tasks/operations.py

```diff
@@ -80,7 +93,31 @@ def __init__(self, local):
     @asyncio.coroutine
     def run(self):
         print("Delete Local File: {}".format(self.local))
-
+        db_parent = session.query(models.File).filter(models.File.osf_id == self.local.parent.id).one()
+        path = os.path.join(db_parent.path, self.local.name)
+        if not path:
```

Comment:
Under what conditions would `path` be undefined?

Try to avoid raising bare exceptions. Consider using a more specific exception type, with a user-readable error message describing the problem.